packages/v4/src/content/design-guidelines/content/brand-voice-and-tone.md

```diff
@@ -0,0 +1,136 @@
+	<td>Collaborative</td>
+</tr>
+	<td>Brave but balanced (not reckless)</td>
+	<td>Inventive</td>
+    </tr>
+</table>
+<br>
+
+You can’t *tell* your users what your voice is. You need to *show* them who you are in your writing. One way to do this is with “Voice Do and Don’t” charts. This is what Red Hat's UX copy voice charts look like:
+
+**Voice trait**: Friendly
+
+**Description**: Our #1 focus is the user. We make them feel welcome and create a sense of belonging and understanding.
+
+<table align="center" style="margin: 0px auto; table-layout:fixed;" tr width="80%">
+    <tr>
+        <th><center>Do</th>
+        <th><center>Don't</th>
+    </tr>
+    <tr>
+        <td>Write how you speak, but add extra polish. Be clear and concise for users of all skills and abilities. Focus on comprehension and use plain language.</td>
+        <td>Don’t include fluff or long sentences that run on and on.</td>
+    </tr>
+</table>
+
+<br> 
+
+**Voice trait**: Approachable
+
+**Description**: People are comfortable engaging with us. We’re open to listening and changing our ways when better ideas come along.
+
+<table align="center" style="margin: 0px auto; table-layout:fixed;" tr width="80%">
+    <tr>
+        <th><center>Do</th>
+        <th><center>Don't</th>
+    </tr>
+    <tr>
+        <td>Say what you mean. Be direct and transparent with descriptive, specific language. </td>
+        <td>Don’t use jargon, idioms, bizspeak, or formal language.</td>
+    </tr>
+</table> 
+
+**Voice trait**: Collaborative
+
+**Description**: We embody Red Hat’s open source mission with our collaborative working style and our sense of community.
+
+<table align="center" style="margin: 0px auto; table-layout:fixed;" tr width="80%">
+    <tr>
+        <th><center>Do</th>
+        <th><center>Don't</th>
+    </tr>
+    <tr>
+        <td>Deliver content in a way that includes everyone. Use “you” and the active voice to put emphasis on the user and power in their hands. </td>
+        <td>Don’t rely on inside jokes, colloquial expressions, culture-specific examples, or other alienating language to get your point across.</td>
+    </tr>
+</table>
+<br>
+Don’t use “I” too much. Make the user the star of every story you tell.
+<br>
+<br>
+
+**Voice trait**: Inventive
+
+**Description**: We have a fearless edge, challenging the assumption that UX is for a niche group of techies. We’re also not afraid to share our ideas and welcome new ones.
+
+<table align="center" style="margin: 0px auto; table-layout:fixed;" tr width="80%">
+    <tr>
+        <th><center>Do</th>
+        <th><center>Don't</th>
+    </tr>
+    <tr>
+        <td>Deliver concepts and ideas with an air of confident simplicity. <br><br>Add real-world, global-friendly examples.</td>
+        <td>Don’t belittle others or make jokes at their expense. We can laugh at ourselves but not at our users. <br><br>Don’t use others as examples of what not to do.</td>
+    </tr>
+</table>
+<br>
```

Comment:
5 more `<br>`s

Reply:
There's 15 in all... I trust you can find them!

Reply:
Haha, I'll find and fix! Thanks!